app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,76 @@
+<p>Appwrite uses emails messages to perform user invite, user verification, login, and reset password. Emails messages can be customized to fit your app's design and voice.</p>
+
+<h2><a href="#customize" id="customize">Customize Templates</a></h2>
+<p>You can customize email templates in the Appwrite console.</p>
+<div class="notice">
+    <h2>Custom SMTP Server Required</h2>
+    <p>The built-in email service does not support custom email templates. Configure a <a href="#smtp">custom SMTP server</a> to enable custom email templates.</p>
+</div>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In your project, navigate to the <b>Auth</b> service.</li>
+    <li>Under the <b>Auth</b> service, navigate to the <b>Templates</b> tab.</li>
+    <li>Expand the email template you want to edit.</li>
+    <li>Select the <b>Template language</b> [TODO: @damodar is this used for localization? Can you configure a different template for each language?].</li>
```

Comment:
@lohanidamodar

Reply:
Yes, that's the idea, each language will have different SMSs and Emails templates